.babelrc

```diff
@@ -1,4 +1,4 @@
 {
-  "presets": ["env", "react"],
+  "presets": ["env", "react", "es2015"],
```

Comment:
what does `es2015` give us that is not already in `react`?

Reply:
It used to fix some vscode error I think, but I'll remove it.